### DISCUSSION TITLE
Need help reading SNES cartridges

### DISCUSSION THREAD

**Discussion by bpm5001** [sanni/cartreader#454] [Need help]
Hi all,

I recently built a v5 and after some troubleshooting and figuring out some other issues I had, I've hit a wall trying to dump a SNES rom. I may be having issues with N64 too, but I don't have a cartridge to test it out at the moment.

When I power on my Cart Reader, I can see the menu on my display and I can select Super Nintendo->Super Nintendo, but then I just get a blank screen after that. I've let it sit for a while to see if it was just taking some time to read the cartridge, but nothing ever happens. I'm stumped to what I am missing.

About my build, I only have the GameBoy, N64 and SNES, slots installed right now. I'm waiting on the NES slot to arrive. Do I need to have all the slots installed for it to work? I was under the impression I could just install the ones I wanted. Also, my status LED doesn't ever light up. My first Mega 2560 pro doesn't work and when I was fiddling with it, the LED would turn on sometimes, but only if some of the pins were not fully seated. I'm not sure if this is related to the issues I'm having with dumping the SNES rom. I'll also admit, I only have one SNES cart right now, but I should be able to grab another one tomorrow and test to see if it is the specific cart. Lastly, I'm running version 9.1 of the firmware.

Any help sorting this out would be much appreciated. Let me know what I can upload to help with figuring out my problem.

**Comment by bpm5001** [comment]
My bad, thought I looked through everything, but found a thread with the same issue already exists at https://github.com/sanni/cartreader/discussions/432

**Comment by sanni** [comment]
Sounds like the database is locking up. You can try deleting snes.txt from the SD card and then see if it behaves differently.

**Reply by bpm5001** [reply]
Removed the snes.txt file from the SD card, same result. I looked through the other thread and tried a few things. First, I was using a linux distro to copy the files over, so I changed to using Windows as you suggested to the other person, but still the same result. I'm going to go through with a multimeter and check to see if there is a bad solder joint somewhere. Concerned it's something more than the firmware since the status LED doesn't light up. Also, I'll have a GBA cart and N64 cart tomorrow and I'll check to see if either of those works.

**Reply by bpm5001** [reply]
So I went through everything again and realized I did something real dumb. I soldered the pins to the wrong side of the board for the clock generator, so everything from there was reversed. 🤦 😁 I guess I was nervous since it was the first thing I soldered and this project is the first soldering project I've done in a long time. I wasn't paying attention.

I still don't have the status LED, but that's less of a big deal.

**Reply by bpm5001** [reply]
Again, nevermind, double checked when reading my SNES cart again, and the LED does turn on, so I think everything is good now. Excited to try out the other other games I have coming today.
